Automattic/wp-calypso
# Editor: Reset Post Edits when switching posts in the post editor

If we create a new post, switch to a draft from the sidebar without saving and coming back to /post/new again, some initial post edits were kept instead of having a fresh new post. Currently, we can experience this for the "sticky" post flag for example. 

While we continue reduxifying the editor, this behaviour will be more and more troublesome. (see #8814 )

This PR resets the postEdits while switching posts on the PostEditor Component.

**Testing instructions**

 - Start new post
 - Click on the "sticky" post icon
 - Before post saves, open drafts drawer and select another post
 - Accept warning that changes will be lost
 - Observe draft loads correctly
 - Start new post
 - The "sticky" post icon should not be selected, we should have a fresh post
